pkg/eks/eks.go

```diff
@@ -66,6 +68,49 @@ func (c *ClusterProvider) RefreshClusterStatus(spec *api.ClusterConfig) error {
 	}
 }
 
+// VersionSupportsManagedNodes reports whether an existing cluster supports Managed Nodes
```

Comment:
`VersionSupportsManagedNodes ` doesn't match `SupportsManagedNodes`, I suspect `lint` will fail once the build get fixed?

Reply:
Good catch (looks like the IDE failed to rename this when I refactored it)